daemon/src/position_metrics.rs

```diff
@@ -377,30 +382,35 @@ mod metrics {
     const STATUS_REJECTED_LABEL: &str = "rejected";
     const STATUS_REFUNDED_LABEL: &str = "refunded";
 
+    const SYMBOL_LABEL: &str = "symbol";
+    pub const BTCUSD_LABEL: &str = "BTCUSD";
```

Comment:
maybe for consistency with broader financial system we could use `XBTUSD`?

Reply:
I think it's pretty random if exchange use `xbtusd` or `btcusd`